### PR TITLE
Fixes #16846: tests changes to prepare rudder-lang integration

### PR DESCRIPTION
**Pull request by gpoblon**
https://issues.rudder.io/issues/16846

wip preparing production test loop to work out of the box hopefully

### CODE REVIEW

**Comment by peckpeck** [comment]
You need a rebase here

File: rudder-lang/tools/cfjson_tester [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+Usage: 
+    ncf ncf-to-json <file.cf> <new.json> [<local_config_path.conf>]
+    ncf compare-json <first.json> <new.json> [<local_config_path.conf>]
+    ncf compare-cf <first.cf> <new.cf> [<local_config_path.conf>]
+"""
+
+import sys
+import os
+import configparser
```
Comment (peckpeck) [inline rudder-lang/tools/cfjson_tester]: this should not work on python2 https://docs.python.org/2/library/configparser.html

**Comment by gpoblon** [comment]
PR updated with a new commit

File: rudder-lang/tools/tester-dev.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (peckpeck) [reply rudder-lang/tools/tester-dev.sh]: ?

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**